apps/comps/components/leaderboard-table/index.tsx

```diff
@@ -0,0 +1,202 @@
+                  sortState={sortState["name"]}
+                >
+                  Agent
+                </SortableTableHeader>
+                <SortableTableHeader
+                  className="flex justify-end text-white"
+                  onToggleSort={() => handleSortChange("competitions")}
+                  sortState={sortState["competitions"]}
+                >
+                  <div className="w-10 sm:w-full">
+                    <p className="truncate">Competitions</p>
+                  </div>
+                </SortableTableHeader>
+                <SortableTableHeader
+                  className="flex justify-end text-white"
+                  onToggleSort={() => handleSortChange("votes")}
+                  sortState={sortState["votes"]}
+                >
+                  Votes
+                </SortableTableHeader>
+                <TableHead className="flex justify-end pr-10 text-white">
+                  Profile
+                </TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {agents.map((agent) => {
+                const scoreSplit = agent.score.toString().split(".");
+                const scoreSize =
+                  scoreSplit.length > 0 ? scoreSplit[1]?.length || 0 : 0;
+
+                return (
+                  <TableRow
+                    key={agent.id}
+                    className="grid grid-cols-[1fr_1fr_2fr_1fr_1fr_1fr]"
+                  >
+                    <TableCell className="xs:flex-row mr-10 flex flex-col items-center gap-2 py-6 pl-20">
+                      {agent.rank === 1 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-[#594100] p-2 text-yellow-500",
+                          )}
+                        >
+                          <Trophy size={17} />
+                          <span>1st</span>
+                        </div>
+                      ) : agent.rank === 2 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-gray-700 p-2 text-gray-300",
+                          )}
+                        >
+                          <AwardIcon size={17} />
+                          <span>2nd</span>
+                        </div>
+                      ) : agent.rank === 3 ? (
+                        <div
+                          className={cn(
+                            "flex w-20 items-center justify-center gap-1 rounded bg-[#1A0E05] p-2 text-[#C76E29]",
+                          )}
+                        >
+                          <AwardIcon size={17} />
+                          <span>3rd</span>
+                        </div>
+                      ) : (
+                        <div className="ml-7 flex items-center justify-center rounded bg-gray-800 p-2">
```

Comment:
```suggestion
                        <div className="flex w-20 items-center justify-center rounded bg-gray-800 p-2">
```

Reply:
the quicket fix is to do what the competitions leaderboard page does and keep the div width the same for all numbers. this doesnt perfectly math the design, but it'll fix the spacing issue that i'm still seeing

Reply:
- without this change
<img width="1323" alt="Screenshot 2025-06-19 at 9 54 40 AM" src="https://github.com/user-attachments/assets/47ed757d-94c2-4ae6-bf4c-80981971b406" />

- with the recommended change
<img width="1313" alt="Screenshot 2025-06-19 at 9 54 12 AM" src="https://github.com/user-attachments/assets/6619450b-8ba0-42c4-a946-26f18fd81081" />